src/pgen/cluster/agn_triggering.cpp

```diff
@@ -122,11 +134,14 @@ AGNTriggering::AGNTriggering(parthenon::ParameterInput *pin,
 // and simultaneously remove cold gas (updating conserveds and primitives)
 template <typename EOS>
 void AGNTriggering::ReduceColdMass(parthenon::Real &cold_mass,
+                                   parthenon::Real &total_mass,
                                    parthenon::MeshData<parthenon::Real> *md,
-                                   const parthenon::Real dt, const EOS eos) const {
```

Comment:
I think the `const` can stay as the function does not modify the member variables of the `AGNTriggering` object.